server/service/src/sync/program_schemas/patient.json

```diff
@@ -376,8 +361,6 @@
         },
         "birthOrder": {
           "description": "Birth order, e.g. \"02\" for second child",
-          "examples": ["01", "02", "03"],
-          "pattern": "^[0-9]{2}$",
```

Comment:
No longer required.

Reply:
probably overkill but you can add "maximum": 99, to have two digits